plugins/org.eclipse.osee.orcs.db/src/org/eclipse/osee/orcs/db/internal/sql/SelectiveArtifactSqlWriter.java

```diff
@@ -457,11 +420,59 @@ private void writeRelsCommonTableExpression(String artWithAlias) {
       String primary = "primaryArt";
       String secondary = "secondaryArt";
       String txdAlias = "txd";
+      String tuple2Alias = "t2";
+      String tuple2TxsAlias = "t2Txs";
+      writeUseNlTableHint(relAlias + " " + relTxsAlias);
+      writeSelectFields(primary, "*", relAlias, "rel_link_type_id AS type_id");
+      write(
+         ", 'B'  AS value, '' AS spare1, 0 AS spare2 ,%s.gamma_id, b_art_id AS other_art_id, %s.art_type_id as other_art_type_id, %s.gamma_id as other_art_gamma_id, 0 as rel_type, 0 as rel_order",
```

Comment:
Can we make sure to leave a space at the end of the string, I don't know if writeSelectFields handles it below

Reply:
Looks like it does...but I can add anyways